src/nuget.config

```diff
@@ -1,5 +1,9 @@
 <configuration>
   <config>
-    <add key="repositoryPath" value="../lib" />
+    <add key="repositoryPath" value="../packages" />
```

Comment:
packages is the default folder name used by nuget. Yes, we can change it, but why not do what VS does by default as it makes it that much more recognizable by anyone else in the OSS community that may look at the project or try to build it? And "packages" is already skipped by the .gitignore file too.

By including the `../` in the path, it places it _outside_ the src folder. Which is great because it leaves src for source.
But it makes little or no difference, since we're using project.json instead of packages.config and as such, you don't get a packages folder at all anyway by default.